test/db_table_creation.txt

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE hosts (
+    host_id SERIAL PRIMARY KEY,
+    host_name VARCHAR(100) UNIQUE NOT NULL
+);
+
+CREATE TABLE users (
+    user_id SERIAL PRIMARY KEY,
+	host_id INTEGER REFERENCES hosts(host_id),
+    user_name VARCHAR(100) NOT NULL,
+    password_hash TEXT NOT NULL,
```

Comment:
Depends on the selected hash algoritms this field needs up to 64 bytes (for SHA512). You could use `binary(64)` or `varbinary(64)` types here.

Reply:
We use Argon2i algorithm to hash passwords. Hashed password size is 128 bytes. We use ```text``` to store hash because it is more flexible. We can use ```varchar(n)``` or ```char(n)``` instead of ```text```, but under the hood in PostgreSQL they are the same. If it is needed or if it is better, we can store hashed passwords in ```bytea```.